apps/storybook/src/data-display/Disclosure.stories.tsx

```diff
@@ -48,3 +49,17 @@ export const ChevronPosition: Story = {
     </Flex>
   ),
 };
+
+export const Controlled: Story = {
+  render: function DefaultSelected(args) {
```

Comment:
rename this to a generic name like Story or Sample or something related to this story like Controlled
```suggestion
  render: function Controlled(args) {
```